### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by SmuSmu**
#74

## Description
German Translation

## PR Type
*This pull request is a/deals with...*
- [ ] Fix (Bug/Security/Otherwise)
- [x] Optimization
- [ ] Documentation Update
- [ ] Enhancement/Feature Addition
- [ ] Other

## PR Category
*This pull request modifies at least one of the following...*
- [ ] Server Backend (`/src/`)
- [ ] Curriculum Files (`/static/curriculum/`)
- [x] Website Front End (`/views/`, `/static/css/`, `/static/js/`)
- [ ] Editor (`/views/editor.ejs`, `/static/js/editor.js`)
- [ ] Other Static Files (`/static/`)
- [ ] Organization/Dependencies/Quality of Life/Other

## Misc.
*By creating this pull request you agree to the following...*
- [x] I have read the [Ideoxan Repository Code of Conduct](https://github.com/ideoxan/ideoxan/blob/master/CODE_OF_CONDUCT.md)
- [x] I have read the [Ideoxan Repository Contributing Guidelines](https://github.com/ideoxan/ideoxan/blob/master/CONTRIBUTING.md)
- [x] I read and agree to the [Terms and Conditions](https://ideoxan.com/tos) and [Privacy Policy](https://ideoxan.com/privacy) of Ideoxan

### CODE REVIEW

**Comment by Skyclo** [comment]
Wow that was fast 😅

**Comment by SmuSmu** [comment]
> Wow that was fast 😅
🤘

**Comment by SmuSmu** [comment]
Hi. You Need to add hacktoberfest-accepted Label to this pr. Or hacktoberfest to the repository

**Comment by Skyclo** [comment]
> Hi. You Need to add hacktoberfest-accepted Label to this pr. Or hacktoberfest to the repository

The Hacktoberfest label is already added to all eligible Ideoxan repositories

**Comment by SmuSmu** [comment]
Strange Wonder why IT was mit counted? Mhh maybe IT needs some time extra?

**Comment by Skyclo** [comment]
> Strange Wonder why IT was mit counted? Mhh maybe IT needs some time extra?

It either needs to be reviewed or merged for it to count. If everything works, I will review it and merge it.

File: content/www/locales/de.json [hunk 1]
```diff
@@ -2,246 +2,246 @@
...
         "features": {
-            "header": "Features",
-            "subheader": "Ideoxan provides content rich features and courses",
+            "header": "Funktionen",
+            "subheader": "Ideoxan bietet inhaltsreiche Funktionen und Kurse",
             "items": {
                 "feature0": {
                     "title": "Open Source",
-                    "description": "All of our content is open to be modified and used by anyone. Our GitHub holds all of the code used to power the website and the courses. These courses can be used in any environment as they are licensed under the Creative Commons Attribution-ShareAlike 4.0 License"
+                    "description": "AAlle unsere Inhalte können von jedermann geändert und verwendet werden. Unser GitHub enthält den gesamten Code, mit dem die Website und die Kurse betrieben werden. Diese Kurse können in jeder Umgebung verwendet werden, da sie unter der Creative Commons Attribution-ShareAlike 4.0-Lizenz lizenziert sind"
```
Comment (Skyclo) [inline content/www/locales/de.json:10]: Is this supposed to be `Alle` instead of `AAlle`?

File: content/www/locales/de.json [hunk 2]
```diff
@@ -2,246 +2,246 @@
...
+            "numberOfChapters": "Anzahl Kapitel",
+            "numberOfLessons": "Anzahl Lektionen",
             "status": {
                 "name": "Status",
-                "comingSoon": "Coming Soon",
-                "free": "Free"
+                "comingSoon": "Demnächst",
+                "free": "frei"
             },
             "sourceCode": "Source Code"
```
Comment (Skyclo) [inline content/www/locales/de.json:231]: Does this have a translation?

**Review by Skyclo — COMMENTED** [review]
I have added some questions. Other than those concerns, the PR is ready to go!

**Comment by SmuSmu** [comment]
Thanks for review. fixed the Aalle Typo and made the translation for sourcecode

**Review by Skyclo — APPROVED** [review]

**Pull request merged**